src/adapter/src/catalog/builtin.rs

```diff
@@ -2036,6 +2036,38 @@ AS SELECT
 WHERE false",
 };
 
+pub const PG_AUTHID: BuiltinView = BuiltinView {
+    name: "pg_authid",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_catalog.pg_authid
+AS SELECT
+    r.oid AS oid,
+    r.name AS name,
```

Comment:
As per https://www.postgresql.org/docs/current/catalog-pg-authid.html this should be named `rolname`

Reply:
Thanks, just fixed this.